canal/canal.go

```diff
@@ -11,13 +11,13 @@ import (
 	"sync"
 	"time"
 
+	"github.com/bytewatch/dolphinbeat/schema"
```

Comment:
Em, can you put all to go-mysql and not import any dolphinbeat? It is very strange for circular dependence. E.g, go-mysql depends dolphinebeat, and dolphinebeat depends on go-mysql.

Reply:
I have copied the source from dolpinbeat/schema to go-mysql/schema. Now I have a build problem under go 1.9 becuase pingcap/parser depends on `strings.Builder`:   [Build 655](https://travis-ci.org/siddontang/go-mysql/jobs/482442176). Do you have any idea?

Reply:
It's ok to build now, PTAL.  By the way, can i ask a question about TiDB's Percolator implemention? I wonder can it turn to 1PC if all mutations is in one region to improve perfermance?